compiler/env/OMRCompilerEnv.hpp

```diff
@@ -97,7 +98,12 @@ class OMR_EXTENSIBLE CompilerEnv
    // Debug environment for the compiler.  This is not thread safe.
    //
    TR::DebugEnv debug;
+   TR::AOTAdapter* aotAdapter;
 
+   // Set the AOT object
```

Comment:
This comment isn't useful; I would remove it.

Reply:
Thank you for the comment. I've also got rid of the setter!